src/igmpproxy.c

```diff
@@ -140,16 +143,42 @@ int main( int ArgCn, char *ArgVc[] ) {
             break;
         }
 
+        // Open /dev/null before chrooting.
+        if (!NotAsDaemon) {
+            devnull = open(_PATH_DEVNULL, O_RDWR, 0);
```

Comment:
`_PATH_DEVNULL` does not have to be available, so stick with `"/dev/null"` as it was before your change.

Reply:
Done.